bundles/sirix-core/src/test/resources/json/test.json

```diff
@@ -1 +1 @@
-{"foo":["bar",null,2.33],"bar":{"hello":"world","helloo":true},"baz":"hello","tada":[{"foo":"bar"},{"baz":false},"boo",{},[]]}
\ No newline at end of file
+{"foo":["bar",null,2.33],"bar":{"hello":"world","helloo":true},"baz":"hello","tada":[{"foo":"bar"},{"baz":false},"boo",[],{}]}
```

Comment:
Is this really the case, that the empty object and empty array changed the position?

Reply:
Yes, I've changed it for remove last child as array test

Reply:
Ah okay 👍